invenio_orcid/utils.py

```diff
@@ -31,3 +31,30 @@ def get_authors_credentials(orcid):
         id=orcid, method='orcid').first()
     user = RemoteAccount.query.filter_by(user_id=raw_user.id_user).first()
     return user.tokens[0].access_token
+
+
+def convert_to_orcid(record):
```

Comment:
OK. What if I provide a simple example record in MARC with just title and DOI, and provide an implementation (written in pure python) that transform the dictionary representation of MARC21 to the dictionary representation of orcid? (using dojson would be overkill for the sake of an example)

MARC example:
```xml
<record>
    <datafield tag="024" ind1="7" ind2=" ">
        <subfield code="a">10.1234/foo.bar</subfield>
        <subfield code="2">DOI</subfield>
    </datafield>
    <datafield tag="245" ind1=" " ind2=" ">
        <subfield code="a">The title</subfield>
    </datafield>
</record>
```